### ISSUE TITLE
Cant get rid of Curse of Binding items

### ISSUE THREAD

**Issue by EinsWlanKabel** [henkelmax/gravestone#120]
**Is your feature request related to a problem? Please describe.**
break_pickup equips any curse of binding armor items.

**Describe the solution you'd like**
sorting on pickup to not equip curse of binding armor or a config option for it.

**Describe alternatives you've considered**
maybe just drop curse of binding items at death.

**Comment by FredMcFrizzle** [comment]
a feature like holding shift while breaking your grave makes it drop on the ground or not equip armour would be a good way around it too

**Comment by ipaqmaster** [comment]
Experiencing this in Direwolf20 1.21 (v1.3.0).

I tried everything including filling my inventory and armor slots so there was no room left.  Gravestone Mod insisted on dropping those items to forcefully re-equip the problematic armor. Serious issue. There needs to be a way to get your items back without the mod automatically unequipping other armor to put cursed items back on.

**Comment by zperk13** [comment]
I agree this should be a thing, but for now, you can get around this by setting break_pickup to false in the config
